### DISCUSSION TITLE
Can SeleniumBase Hide WebDriver from Detection?

### DISCUSSION THREAD

**Discussion by amirzarbakhsh** [seleniumbase/SeleniumBase#3286] [Q&A] [answered]
Hello,

I’ve read in multiple documents that SeleniumBase has features to bypass anti-bot systems. My goal is to create a tool for generating fake visits to websites, and it is crucial for me that neither Google nor similar platforms can detect that the browser I’m using is a WebDriver.

Does SeleniumBase have the capability to completely hide the WebDriver and make it indistinguishable from a regular browser? If yes, I would appreciate it if you could guide me on how to implement this functionality.

Thank you!

**Comment by mdmintz** [comment, marked as answer]
See the [**CDP Mode** ReadMe](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/cdp_mode/ReadMe.md) for hiding automation from detection systems.
